decoupled-site/frontend/codegen.yaml

```diff
@@ -1,4 +1,4 @@
-schema: https://cg.optimizely.com/content/v2?experimental=true&auth=INPUT_SINGLE_KEY_HERE
+schema: ${CONTENT_GRAPH_ENDPOINT}
```

Comment:
I think we should revert this change because our customer would only use this endpoint to consistent with our docs and Readme file

Reply:
this should be configured from env file or environment variable of hosting like vercel. The purpose is we can use our key.
<img width="971" alt="image" src="https://user-images.githubusercontent.com/8802680/223902337-11c57388-ab46-438a-b2c5-8a30844f837c.png">